src/docs/Components/Form/Autocomplete/previews/multi.preview.html

```diff
@@ -47,7 +47,7 @@
       <input
         id="checkbox-01"
         type="checkbox"
-        class="mc-listbox__input"
```

Comment:
Both classes should be put on the element: `class="mc-listbox__input mc-checkbox__input"`